Python/Product/Analysis/Values/BuiltinNamespace.cs

```diff
@@ -133,6 +133,6 @@ class BuiltinNamespace<MemberContainerType> : AnalysisValue where MemberContaine
             return false;
         }
 
-        public override int GetHashCode() => new { hc1 = GetType().GetHashCode(), hc2 = _type.GetHashCode() }.GetHashCode();
+        public override int GetHashCode() => new { hc1 = GetType()?.GetHashCode() ?? 0, hc2 = _type?.GetHashCode() ?? 0 }.GetHashCode();
```

Comment:
Think I just saw a slightly different fix in another PR, but now I look at it, we shouldn't have to call `GetHashCode()` on the members here. The anonymous type `GetHashCode` will do it (and handle nulls properly)

Reply:
Yes, will merge